phobos/core/robot.py

```diff
@@ -108,13 +110,16 @@ def get_robot_from_blender_dict(cls, name='', objectlist=[], blender_model=None)
                 ))
         mats = []
         for key, value in blender_model['materials'].items():
-            # TODO internal dict hat mehr als eine Möglichkeit für Color
-            mats.append(representation.Material(name=value.get('name'),
-                                                texture=None,
-                                                diffuseColor=value.get("diffuseColor")
+            diff_color = value.pop('diffuseColor')
+            mats.append(representation.Material(name=value.pop('name'),
+                                                color=representation.Color(diff_color["r"],
+                                                                           diff_color["g"],
+                                                                           diff_color["b"],
+                                                                           texture=None),
+                                                **value
```

Comment:
This change is obsolete The removed lines are clearer, but please add the other Material's colors here as well